cookiecutter.json

```diff
@@ -1,31 +1,36 @@
 {
-  "full_name": "<Your full name>",
-  "email": "<Your email>",
-  "contribution_name": "<Contribution name>",
-  "autora_contribution_type": ["theorist", "experimentalist", "experiment_runner", "synthetic_data", "custom"],
-  "custom_autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}<custom_autora_contribution_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "full_name": "Enter your full name",
+  "email": "Enter your e-mail adress",
+  "contribution_name": "Choose a name of your contribution here. This will also be your package name. Choose something short but descriptive",
+  "__contrib_name": "{% if cookiecutter.contribution_name == 'Choose a name of your contribution here. This will also be your package name. Choose something short but descriptive' -%}{% else -%}{{ cookiecutter.contribution_name }}{% endif -%}",
+  "project_short_description": "Describe your contriution",
+  "repository": "If you have already created a GitHub repository for your package, you can paste in the url to here",
+  "__repository": "{% if cookiecutter.repository == 'If you have already created a GitHub repository for your package, you can paste in the url to here' -%}{% else -%}{{ cookiecutter.repository }}{% endif -%}",
```

Comment:
```suggestion
  "__repository": "{% if cookiecutter.repository == 'If you have already created a GitHub repository for your package, paste the url here. Otherwise, press Enter to skip' -%}{% else -%}{{ cookiecutter.repository }}{% endif -%}",
```